docs/sphinx/_toc.yml.in

```diff
@@ -41,7 +41,7 @@ subtrees:
     - file: reference/rocm-tools.md
       title: Tools
     - file: reference/gpu-arch-specs.rst
-      title: Hardware specifications
+      title: Accelerator & GPU hardware specifications
```

Comment:
Personally, I liked "Hardware specifications".  It was shorter and fit in the left pane.

Reply:
That's very true. I thought it might sound a little too vague. Can we compromise and say "Accelerator & GPU specifications"?

It's precise and still fits in the sidebar. Maybe "hardware" is obvious.

![image](https://github.com/ROCm/ROCm/assets/115042610/f4ee709f-60fd-46bb-8510-abfd32dc5548)